back-end/db/host.ts

```diff
@@ -18,9 +18,7 @@ export default {
 
   hostOpenGame: async function(socketId: string): Promise<number> {
     try {
-      const allGameIds = await this.getAllGameIds();
-      const maxId = allGameIds.length && Math.max(...allGameIds);
-      const newId = maxId+1;
+      const newId = Math.floor(Math.random() * 9000 + 1000);
```

Comment:
It's very unlikely, but we will want to make sure this generated ID is not already claimed by another game